domains/csozi.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "csozi",
+    "email": "csozi.dev@gmail.com"
+  },
+  "record": {
+    "CNAME": "85.202.160.143"
```

CNAME should be a domain name, did you mean A record?
```suggestion
    "A": ["85.202.160.143"]
```